src/hardware/RGBLEDstrip.cpp

```diff
@@ -0,0 +1,107 @@
+/**
+ * @file RGBLEDstrip.cpp
+ * 
+ * @brief A strip with adressable LEDs connected to a GPIO pin
+ */
+
+#include "RGBLEDstrip.h"
+#include "chipsets.h"
+#include "Logger.h"
```

Comment:
Include is never used. And those classes shouldn't have a dependency for the logger anyway.